designs/administrator/future-openshift/air-gapped/index.md

```diff
@@ -0,0 +1,32 @@
+---
+parent: Administrator
+version: 4.x
+---
+
+# Configuring Air Gapped Environments
+This unit of work is a part of our continuous effort to improve the upgrade experience. Updates can be configured to receive information from Red Hat or via a custom update service. This set of enhancements allows customers to edit the Cincinnati server location directly in the console UI. 
+
+## 01 | Adding the ability to edit the Cincinnati server location via console to Cluster Settings details
+![1](img/1.png)
+- Add a new **Upstream configuration** field to the **Cluster Settings details** page and allow users to edit the configuration.
+
+![2](img/2.png)
```

Comment:
This is also nit level but maybe the values of radios could read a similar format to each other like 
(o) Default update service from Red Hat
( ) Custom update service:  <-not sure we do colons there
[http://                    ]

I think the periods threw me off a little but also not really a big deal.

Reply:
+1 I am also wondering if default should be first here. Could it be appended like
`Receive update information from Red Hat (default)`

Reply:
I made a few updates - let me know what you think!